include/gatekeeper_main.h

```diff
@@ -47,11 +41,14 @@ extern int gatekeeper_logtype;
 	} while (0)
 
 #define G_LOG(level, ...)		        \
-	rte_log_ratelimit(RTE_LOG_ ## level,	\
-	gatekeeper_logtype, "GATEKEEPER: " __VA_ARGS__)
+	do {								\
+		rte_log_set_level(BLOCK_LOGTYPE, RTE_LOG_DEBUG);	\
```

Comment:
We should set the log level of `BLOCK_LOGTYPE` during initialization instead of per log entry. Moreover, the return of `rte_log_set_level()` must be checked for errors.